obspy/core/event/radpattern.py

```diff
@@ -0,0 +1,549 @@
+# -*- coding: utf-8 -*-
+# -------------------------------------------------------------------
+# Filename: radpattern.py
+#  Purpose: Computes and plots radiation patterns
+# ---------------------------------------------------------------------
```

Comment:
Can you please remove the 4 preceding lines? Any documentation should go in the file's main docstring.